### ISSUE TITLE
Textarea field causes an issue expecting an array, null given

### ISSUE THREAD

**Issue by dgallinari** [formers/former#516]
Stack Trace:
```
ErrorException in /home/vagrant/Code/partners/vendor/laravel/framework/src/Illuminate/Support/Arr.php line 145:
array_key_exists() expects parameter 2 to be array, null given (View: /home/vagrant/Code/partners/resources/views/internal/integrations/partials/banners_table.blade.php) (View: /home/vagrant/Code/partners/resources/views/internal/integrations/partials/banners_table.blade.php)
in Arr.php line 145
at CompilerEngine->handleViewException(object(ErrorException), '1') in PhpEngine.php line 44
at PhpEngine->evaluatePath('/home/vagrant/Code/partners/storage/framework/views/76edf8de7baeef56353827e3414f94a344f826d7.php', array('__env' => object(Factory), 'app' => object(Application), 'errors' => object(ViewErrorBag), 'PartnersInternalMenu' => object(Builder), 'UserActionsMenu' => object(Builder), 'banners' => object(LengthAwarePaginator), 'filters' => array('category' => array('' => 'All', 'Africa Do Sul' => 'Africa Do Sul', 'Canada' => 'Canada', 'Estados Unidos' => 'Estados Unidos', 'Institucional' => 'Institucional', 'Irlanda' => 'Irlanda', 'Malta' => 'Malta'), 'size' => array('' => 'All', '160x600' => '160x600', '240x400' => '240x400', '250x250' => '250x250', '300x600' => '300x600', '728x90' => '728x90')), 'currentUser' => object(User), 'affiliateId' => null, 'am_fill_color_awaiting_approval' => '#FEEF6D', 'am_fill_color_approved' => '#25C8D1', 'am_fill_color_processing_payout' => '#0073b7', 'am_fill_color_paid' => '#74AE34', 'am_fill_color_cancelled' => '#DD4B39', 'am_fill_color_quantity' => '#333C4E')) in CompilerEngine.php line 59
at CompilerEngine->get('/home/vagrant/Code/partners/resources/views/internal/integrations/banners.blade.php', array('__env' => object(Factory), 'app' => object(Application), 'errors' => object(ViewErrorBag), 'PartnersInternalMenu' => object(Builder), 'UserActionsMenu' => object(Builder), 'banners' => object(LengthAwarePaginator), 'filters' => array('category' => array('' => 'All', 'Africa Do Sul' => 'Africa Do Sul', 'Canada' => 'Canada', 'Estados Unidos' => 'Estados Unidos', 'Institucional' => 'Institucional', 'Irlanda' => 'Irlanda', 'Malta' => 'Malta'), 'size' => array('' => 'All', '160x600' => '160x600', '240x400' => '240x400', '250x250' => '250x250', '300x600' => '300x600', '728x90' => '728x90')), 'currentUser' => object(User), 'affiliateId' => null, 'am_fill_color_awaiting_approval' => '#FEEF6D', 'am_fill_color_approved' => '#25C8D1', 'am_fill_color_processing_payout' => '#0073b7', 'am_fill_color_paid' => '#74AE34', 'am_fill_color_cancelled' => '#DD4B39', 'am_fill_color_quantity' => '#333C4E')) in View.php line 147
at View->getContents() in View.php line 118
at View->renderContents() in View.php line 83
at View->render() in Response.php line 53
at Response->setContent(object(View)) in Response.php line 199
at Response->__construct(object(View)) in Router.php line 1087
at Router->prepareResponse(object(Request), object(View)) in ControllerDispatcher.php line 94
at ControllerDispatcher->Illuminate\Routing\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 52
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckPermission.php line 20
at CheckPermission->handle(object(Request), object(Closure), 'app_partners.integrations.banner')
at call_user_func_array(array(object(CheckPermission), 'handle'), array(object(Request), object(Closure), 'app_partners.integrations.banner')) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckPermission.php line 20
at CheckPermission->handle(object(Request), object(Closure), 'apps.partners')
at call_user_func_array(array(object(CheckPermission), 'handle'), array(object(Request), object(Closure), 'apps.partners')) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Authenticate.php line 29
at Authenticate->handle(object(Request), object(Closure))
at call_user_func_array(array(object(Authenticate), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 102
at Pipeline->then(object(Closure)) in ControllerDispatcher.php line 96
at ControllerDispatcher->callWithinStack(object(IntegrationsController), object(Route), object(Request), 'banners') in ControllerDispatcher.php line 54
at ControllerDispatcher->dispatch(object(Route), object(Request), 'Partners\Http\Controllers\Web\Internal\IntegrationsController', 'banners') in Route.php line 174
at Route->runController(object(Request)) in Route.php line 140
at Route->run(object(Request)) in Router.php line 724
at Router->Illuminate\Routing\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 52
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in MenusMiddleware.php line 20
at MenusMiddleware->handle(object(Request), object(Closure))
at call_user_func_array(array(object(MenusMiddleware), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in VerifyCsrfToken.php line 64
at VerifyCsrfToken->handle(object(Request), object(Closure))
at call_user_func_array(array(object(VerifyCsrfToken), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in ShareErrorsFromSession.php line 49
at ShareErrorsFromSession->handle(object(Request), object(Closure))
at call_user_func_array(array(object(ShareErrorsFromSession), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in StartSession.php line 62
at StartSession->handle(object(Request), object(Closure))
at call_user_func_array(array(object(StartSession), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in AddQueuedCookiesToResponse.php line 37
at AddQueuedCookiesToResponse->handle(object(Request), object(Closure))
at call_user_func_array(array(object(AddQueuedCookiesToResponse), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in EncryptCookies.php line 59
at EncryptCookies->handle(object(Request), object(Closure))
at call_user_func_array(array(object(EncryptCookies), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 102
at Pipeline->then(object(Closure)) in Router.php line 726
at Router->runRouteWithinStack(object(Route), object(Request)) in Router.php line 699
at Router->dispatchToRoute(object(Request)) in Router.php line 675
at Router->dispatch(object(Request)) in Kernel.php line 246
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 52
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Debugbar.php line 49
at Debugbar->handle(object(Request), object(Closure))
at call_user_func_array(array(object(Debugbar), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckForMaintenanceMode.php line 44
at CheckForMaintenanceMode->handle(object(Request), object(Closure))
at call_user_func_array(array(object(CheckForMaintenanceMode), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 102
at Pipeline->then(object(Closure)) in Kernel.php line 132
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 99
at Kernel->handle(object(Request)) in index.php line 53```

**Comment by dgallinari** [comment]
@vlakoff

**Comment by dgallinari** [comment]
The textarea field in the view:

```
{!! \Former::textarea('code_snippet') !!}
```

I am using Laravel 5.2.23 and Former 4.0.2.

**Comment by vlakoff** [comment]
The View engine makes it hard, because of the file inclusion mechanism it skips the relevant part of the stack.

To narrow the window, could you temporarily add this at the top of the `vendor/laravel/framework/src/Illuminate/Support/Arr.php` function `get` and see what this reports:

    if (! $array) {
        dd($key, $default);
    }

**Comment by vlakoff** [comment]
@dgallinari I updated the code snippet.

**Comment by dgallinari** [comment]
Ok doing so

**Comment by dgallinari** [comment]
```
"class"
null
```

**Comment by dgallinari** [comment]
@vlakoff The output shows a $key with value "class" and the $default with value NULL.

**Comment by vlakoff** [comment]
Aren't you using ZurbFoundation? My first guess is that `error_classes` might have been incorrectly edited.

**Comment by dgallinari** [comment]
No, we are using TwitterBootstrap

**Comment by vlakoff** [comment]
Updated the snippet again:

    if (! static::accessible($array)) {
        dd($array, $key, $default);
    }

**Comment by claar** [comment]
@dgallinari Can you please change your code to `{!! \Former::textarea('code_snippet')->wrapAndRender() !!}` ?

When I use wrapAndRender(), I get a second stack trace that's much more useful than the one you posted, that includes the actual Former code path.

**Comment by dgallinari** [comment]
ok

**Comment by vlakoff** [comment]
That would be very useful, indeed ;)

**Comment by vlakoff** [comment]
claar sorry for your mailbox spam ;)

**Comment by dgallinari** [comment]
Stack Trace now:
```
ErrorException in /home/vagrant/Code/partners/vendor/laravel/framework/src/Illuminate/Support/Arr.php line 145:
array_key_exists() expects parameter 2 to be array, null given
in Arr.php line 145
at HandleExceptions->handleError('2', 'array_key_exists() expects parameter 2 to be array, null given', '/home/vagrant/Code/partners/vendor/laravel/framework/src/Illuminate/Support/Arr.php', '145', array('array' => null, 'key' => 'code_snippet'))
at array_key_exists('code_snippet', null) in Arr.php line 145
at Arr::exists(null, 'code_snippet') in Arr.php line 273
at Arr::get(null, 'code_snippet', null) in helpers.php line 171
at array_get(null, 'code_snippet') in Former.php line 478
at Former->getRules('code_snippet') in Field.php line 236
at Field->getRules() in Field.php line 111
at Field->__construct(object(Application), 'textarea', 'code_snippet', '', null, null, null, null) in MethodDispatcher.php line 198
at MethodDispatcher->toFields('textarea', array('code_snippet', '')) in Former.php line 154
at Former->__call('textarea', array('code_snippet', '')) in Facade.php line 221
at Facade::__callStatic('textarea', array('code_snippet', '')) in dd27d9464d3c7f4e66557662e2ef6f87dbc64dd9.php line 26
at include('/home/vagrant/Code/partners/storage/framework/views/dd27d9464d3c7f4e66557662e2ef6f87dbc64dd9.php') in PhpEngine.php line 42
at PhpEngine->evaluatePath('/home/vagrant/Code/partners/storage/framework/views/dd27d9464d3c7f4e66557662e2ef6f87dbc64dd9.php', array('__env' => object(Factory), 'app' => object(Application), 'errors' => object(ViewErrorBag), 'PartnersInternalMenu' => object(Builder), 'UserActionsMenu' => object(Builder), 'obLevel' => '1', 'banners' => object(LengthAwarePaginator), 'filters' => array('category' => array('' => 'All', 'Africa Do Sul' => 'Africa Do Sul', 'Canada' => 'Canada', 'Estados Unidos' => 'Estados Unidos', 'Institucional' => 'Institucional', 'Irlanda' => 'Irlanda', 'Malta' => 'Malta'), 'size' => array('' => 'All', '160x600' => '160x600', '240x400' => '240x400', '250x250' => '250x250', '300x600' => '300x600', '728x90' => '728x90')), 'currentUser' => object(User), 'affiliateId' => null, 'am_fill_color_awaiting_approval' => '#FEEF6D', 'am_fill_color_approved' => '#25C8D1', 'am_fill_color_processing_payout' => '#0073b7', 'am_fill_color_paid' => '#74AE34', 'am_fill_color_cancelled' => '#DD4B39', 'am_fill_color_quantity' => '#333C4E')) in CompilerEngine.php line 59
at CompilerEngine->get('/home/vagrant/Code/partners/resources/views/internal/integrations/partials/banners_table.blade.php', array('__env' => object(Factory), 'app' => object(Application), 'errors' => object(ViewErrorBag), 'PartnersInternalMenu' => object(Builder), 'UserActionsMenu' => object(Builder), 'obLevel' => '1', 'banners' => object(LengthAwarePaginator), 'filters' => array('category' => array('' => 'All', 'Africa Do Sul' => 'Africa Do Sul', 'Canada' => 'Canada', 'Estados Unidos' => 'Estados Unidos', 'Institucional' => 'Institucional', 'Irlanda' => 'Irlanda', 'Malta' => 'Malta'), 'size' => array('' => 'All', '160x600' => '160x600', '240x400' => '240x400', '250x250' => '250x250', '300x600' => '300x600', '728x90' => '728x90')), 'currentUser' => object(User), 'affiliateId' => null, 'am_fill_color_awaiting_approval' => '#FEEF6D', 'am_fill_color_approved' => '#25C8D1', 'am_fill_color_processing_payout' => '#0073b7', 'am_fill_color_paid' => '#74AE34', 'am_fill_color_cancelled' => '#DD4B39', 'am_fill_color_quantity' => '#333C4E')) in View.php line 147
at View->getContents() in View.php line 118
at View->renderContents() in View.php line 83
at View->render() in 76edf8de7baeef56353827e3414f94a344f826d7.php line 36
at include('/home/vagrant/Code/partners/storage/framework/views/76edf8de7baeef56353827e3414f94a344f826d7.php') in PhpEngine.php line 42
at PhpEngine->evaluatePath('/home/vagrant/Code/partners/storage/framework/views/76edf8de7baeef56353827e3414f94a344f826d7.php', array('__env' => object(Factory), 'app' => object(Application), 'errors' => object(ViewErrorBag), 'PartnersInternalMenu' => object(Builder), 'UserActionsMenu' => object(Builder), 'banners' => object(LengthAwarePaginator), 'filters' => array('category' => array('' => 'All', 'Africa Do Sul' => 'Africa Do Sul', 'Canada' => 'Canada', 'Estados Unidos' => 'Estados Unidos', 'Institucional' => 'Institucional', 'Irlanda' => 'Irlanda', 'Malta' => 'Malta'), 'size' => array('' => 'All', '160x600' => '160x600', '240x400' => '240x400', '250x250' => '250x250', '300x600' => '300x600', '728x90' => '728x90')), 'currentUser' => object(User), 'affiliateId' => null, 'am_fill_color_awaiting_approval' => '#FEEF6D', 'am_fill_color_approved' => '#25C8D1', 'am_fill_color_processing_payout' => '#0073b7', 'am_fill_color_paid' => '#74AE34', 'am_fill_color_cancelled' => '#DD4B39', 'am_fill_color_quantity' => '#333C4E')) in CompilerEngine.php line 59
at CompilerEngine->get('/home/vagrant/Code/partners/resources/views/internal/integrations/banners.blade.php', array('__env' => object(Factory), 'app' => object(Application), 'errors' => object(ViewErrorBag), 'PartnersInternalMenu' => object(Builder), 'UserActionsMenu' => object(Builder), 'banners' => object(LengthAwarePaginator), 'filters' => array('category' => array('' => 'All', 'Africa Do Sul' => 'Africa Do Sul', 'Canada' => 'Canada', 'Estados Unidos' => 'Estados Unidos', 'Institucional' => 'Institucional', 'Irlanda' => 'Irlanda', 'Malta' => 'Malta'), 'size' => array('' => 'All', '160x600' => '160x600', '240x400' => '240x400', '250x250' => '250x250', '300x600' => '300x600', '728x90' => '728x90')), 'currentUser' => object(User), 'affiliateId' => null, 'am_fill_color_awaiting_approval' => '#FEEF6D', 'am_fill_color_approved' => '#25C8D1', 'am_fill_color_processing_payout' => '#0073b7', 'am_fill_color_paid' => '#74AE34', 'am_fill_color_cancelled' => '#DD4B39', 'am_fill_color_quantity' => '#333C4E')) in View.php line 147
at View->getContents() in View.php line 118
at View->renderContents() in View.php line 83
at View->render() in Response.php line 53
at Response->setContent(object(View)) in Response.php line 199
at Response->__construct(object(View)) in Router.php line 1087
at Router->prepareResponse(object(Request), object(View)) in ControllerDispatcher.php line 94
at ControllerDispatcher->Illuminate\Routing\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 52
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckPermission.php line 20
at CheckPermission->handle(object(Request), object(Closure), 'app_partners.integrations.banner')
at call_user_func_array(array(object(CheckPermission), 'handle'), array(object(Request), object(Closure), 'app_partners.integrations.banner')) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckPermission.php line 20
at CheckPermission->handle(object(Request), object(Closure), 'apps.partners')
at call_user_func_array(array(object(CheckPermission), 'handle'), array(object(Request), object(Closure), 'apps.partners')) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Authenticate.php line 29
at Authenticate->handle(object(Request), object(Closure))
at call_user_func_array(array(object(Authenticate), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 102
at Pipeline->then(object(Closure)) in ControllerDispatcher.php line 96
at ControllerDispatcher->callWithinStack(object(IntegrationsController), object(Route), object(Request), 'banners') in ControllerDispatcher.php line 54
at ControllerDispatcher->dispatch(object(Route), object(Request), 'Partners\Http\Controllers\Web\Internal\IntegrationsController', 'banners') in Route.php line 174
at Route->runController(object(Request)) in Route.php line 140
at Route->run(object(Request)) in Router.php line 724
at Router->Illuminate\Routing\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 52
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in MenusMiddleware.php line 20
at MenusMiddleware->handle(object(Request), object(Closure))
at call_user_func_array(array(object(MenusMiddleware), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in VerifyCsrfToken.php line 64
at VerifyCsrfToken->handle(object(Request), object(Closure))
at call_user_func_array(array(object(VerifyCsrfToken), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in ShareErrorsFromSession.php line 49
at ShareErrorsFromSession->handle(object(Request), object(Closure))
at call_user_func_array(array(object(ShareErrorsFromSession), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in StartSession.php line 62
at StartSession->handle(object(Request), object(Closure))
at call_user_func_array(array(object(StartSession), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in AddQueuedCookiesToResponse.php line 37
at AddQueuedCookiesToResponse->handle(object(Request), object(Closure))
at call_user_func_array(array(object(AddQueuedCookiesToResponse), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in EncryptCookies.php line 59
at EncryptCookies->handle(object(Request), object(Closure))
at call_user_func_array(array(object(EncryptCookies), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 102
at Pipeline->then(object(Closure)) in Router.php line 726
at Router->runRouteWithinStack(object(Route), object(Request)) in Router.php line 699
at Router->dispatchToRoute(object(Request)) in Router.php line 675
at Router->dispatch(object(Request)) in Kernel.php line 246
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 52
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Debugbar.php line 49
at Debugbar->handle(object(Request), object(Closure))
at call_user_func_array(array(object(Debugbar), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckForMaintenanceMode.php line 44
at CheckForMaintenanceMode->handle(object(Request), object(Closure))
at call_user_func_array(array(object(CheckForMaintenanceMode), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 32
at Pipeline->Illuminate\Routing\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 102
at Pipeline->then(object(Closure)) in Kernel.php line 132
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 99
at Kernel->handle(object(Request)) in index.php line 53
```

**Comment by vlakoff** [comment]
I FOUND IT THE LITTLE BASTARD

**Comment by dgallinari** [comment]
HAHAHAHAHA

**Comment by dgallinari** [comment]
Thanks @vlakoff !! I checked here and this change fixes the issue! :smile: 

I really appreciate your attention and support dude! :D

**Comment by claar** [comment]
Fixed in Former 4.0.3, thanks @vlakoff and @dgallinari
